tests/integration_tests/test_chat_completion.py

```diff
@@ -370,12 +406,6 @@ def dial_recall_expected(r: ChatCompletionResult):
     )
 
     if is_llama3(deployment):
-        test_case(
```

Comment:
what happened with this one?

Reply:
Converse API does not allow cases, when there is `ai` message before `user`

Reply:
this is a valid negative test case then

Reply:
Fixed, added test case with exception